test/tyche/YieldDirector.test.ts

```diff
@@ -47,12 +40,16 @@ describe.only("YieldDirector", async () => {
     // TODO needs cleanup. use Bignumber.
     // Mine block and rebase. Returns the new index.
     const triggerRebase = async () => {
-        mineBlock();
+        await timeout(28800) // 8 hours per epoch
         await staking.rebase();
-
         return await sOhm.index();
     };
 
+    const timeout = async (seconds: number): Promise<void> => {
```

Comment:
Can you rename this method to be still something related to mining blocks. i got it confused with `setTimeout`